node-operators/rpc.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Run an RPC Node Using Docker
+description: How to set up and run a ContainerChain node using Docker, so you can have your own RPC endpoint-as-a-backend to support your DApps and interact with your chain.
+---
+
+# Run an RPC Node Using Docker
+
+## Introduction {: #introduction }
+
+Running an RPC node on a Tanssi ContainerChain allows you to connect to the network, discover other nodes to synch with through a bootnode, and offer an interaction layer for DApps over the WebSocket protocol.
+
+In this guide, you'll learn how to quickly spin up a ContainerChain node using [Docker](https://www.docker.com/){target=\_blank} on a Linux computer.
```

Comment:
```suggestion
In this guide, you'll learn how to quickly spin up a Tanssi Appchain node using [Docker](https://www.docker.com/){target=\_blank} on a Linux computer.
```

Reply:
Suggest that steps can be adapted for Mac?

Reply:
The catch here is that the relay chain specs are not public. Any regular user wouldn't be able to bundle it together with their own chain specs and binaries.
(And we are currently releasing only linux based images)

Reply:
https://drive.google.com/drive/folders/1JVyj518T8a77xKXOBgcBe77EEsjnSFFO - We can put this in a PaperMoon repo if we want